batch/batch/cloud/azure/driver/driver.py

```diff
@@ -56,12 +55,10 @@ async def create(
         with open(os.environ['HAIL_SSH_PUBLIC_KEY'], encoding='utf-8') as f:
             ssh_public_key = f.read()
 
-        arm_client = aioazure.AzureResourceManagerClient(
-            subscription_id, resource_group, credentials_file=credentials_file
-        )
-        compute_client = aioazure.AzureComputeClient(subscription_id, resource_group, credentials_file=credentials_file)
-        resources_client = aioazure.AzureResourcesClient(subscription_id, credentials_file=credentials_file)
-        network_client = aioazure.AzureNetworkClient(subscription_id, resource_group, credentials_file=credentials_file)
```

Comment:
I think we don't *need* to specify `AZURE_APPLICATION_CREDENTIALS` because the `aioazure` credentials pick up on `/gsa-key/key.json` even though `aiogoogle` does not. It's unclear to me how to resolve this asymmetry. Better to specify `AZURE_APPLICATION_CREDENTIALS` for consistency?

Reply:
My inclination is to exclude it rather than confuse folks in the future with "what was this env-var doing, actually?"